src/components/button/button.js

```diff
@@ -40,15 +40,15 @@ export const Button = ({
 Button.propTypes = {
   children: PropTypes.node.isRequired,
   onClick: PropTypes.func.isRequired,
-  type: PropTypes.oneOf(["primary", "danger"]),
+  color: PropTypes.oneOf(["primary", "danger"]),
   size: PropTypes.oneOf(["small", "medium", "large"]),
   disabled: PropTypes.bool,
   inverted: PropTypes.bool,
   inputType: PropTypes.oneOf(["button", "submit", "reset"])
```

Comment:
Please also add a `type` prop to be used as the button's type

Reply:
Thank you for the review!

This component already has a prop called `inputType`. Should I rename it to ｀type｀?

Reply:
Yes, I believe renaming it will make it more intuitive.

Reply:
I will fix it in that way!